src/rust/lib/graph-editor/src/component/node/port.rs

```diff
@@ -237,16 +242,21 @@ impl Manager {
                             def _foo = port.events.mouse_over . map(f_!(hover.set(1.0);));
                             def _foo = port.events.mouse_out  . map(f_!(hover.set(0.0);));
 
-                            def out  = port.events.mouse_out.constant(cursor::Mode::Normal);
-                            def over = port.events.mouse_over.constant(cursor::Mode::highlight(&port,Vector2::new(x,0.0),Vector2::new(width2,height)));
+                            def out  = port.events.mouse_out.constant(cursor::Style::default());
+                            def over = port.events.mouse_over.constant(cursor::Style::highlight(&port,Vector2::new(width2,height),Some(color::Lcha::new(0.6,0.5,0.76,1.0))));
```

Comment:
Line too long

Reply:
its still very dirty file. Ill clean it up soon